include/uv.h

```diff
@@ -243,6 +243,11 @@ typedef enum {
 UV_EXTERN unsigned int uv_version(void);
 UV_EXTERN const char* uv_version_string(void);
 
+typedef void* (*uv_malloc_cb)(size_t size);
+typedef void (*uv_free_cb)(void* ptr);
+
+UV_EXTERN void uv_replace_allocator(uv_malloc_cb malloc_cb, uv_free_cb free_cb);
```

Comment:
As someone who is often "that guy", I totally get it. :-)  I'm happy to use the func prefix.  Will make that change.

Re: a single allocator function for free and malloc, I'm happy to do that too if you don't think it'll be too wonky. It's the kind of thing that would probably require a non-trivial comment explaining it.